libs/pageserver_api/src/key.rs

```diff
@@ -49,6 +51,18 @@ pub const AUX_KEY_PREFIX: u8 = 0x62;
 /// The key prefix of ReplOrigin keys.
 pub const REPL_ORIGIN_KEY_PREFIX: u8 = 0x63;
 
+/// The key prefix of db directory keys.
+pub const DB_DIR_KEY_PREFIX: u8 = 0x64;
+
+/// The key prefix of rel directory keys.
+pub const REL_DIR_KEY_PREFIX: u8 = 0x65;
+
+/// The value of the rel directory keys that indicates the existence of a relation.
+pub const REL_EXISTS_MARKER: Bytes = Bytes::from_static(b"r");
+
+/// A tombstone in the sparse keyspace, which is an empty buffer.
+pub const SPARSE_TOMBSTONE_MARKER: Bytes = Bytes::from_static(b"");
```

Comment:
This is a bit implicit. I'd prefer some sort of enum that we match on instead of checking if the buffer is empty.

Reply:
https://github.com/neondatabase/neon/pull/10593/commits/c7a74e3fbf850b38b4fad993c01bb6f0df2aae15